index.bs

```diff
@@ -245,8 +245,11 @@ For readable streams, an underlying source can use this desired size as a backpr
 generation so as to try to keep the desired size above or at zero. For writable streams, a producer can behave
 similarly, avoiding writes that would cause the desired size to go negative.
 
-Concretely, a queueing strategy is given by any JavaScript object with a <code>highWaterMark</code> property and a
-<code>size()</code> method. <!-- TODO: https://github.com/whatwg/streams/issues/427 -->
+Concretely, a queuing strategy is given by any JavaScript object with a <code>highWaterMark</code> property. For byte
+streams the <code>highWaterMark</code> always has units of bytes. For other streams the default unit is <a>chunks</a>,
+but a <code>size()</code> function can be included in the strategy object which returns the size for a given chunk. This
```

Comment:
Good correction, from method to function.

Reply:
This wasn't actually intentional: I delete the text and rewrote it :-)